services/db.service.js

```diff
@@ -1,12 +1,13 @@
-import { MongoClient } from 'mongodb'
+import { MongoClient, ObjectId } from 'mongodb'
```

Comment:
you dont use here ObjectIf remove the import

Reply:
Yes, we can remove it